### ISSUE TITLE
Fabric MSBuild 1.6.3 MSB3030: Could not copy the file because it was not found

### ISSUE THREAD

**Issue by keggster101020** [Azure/service-fabric-issues#788]
Working with a Service Fabric application and when going to debug/deploy the application in visual studio I receive the error `MSB3030 could not copy the file "<file_path> because it was not found.` 

I have checked the location mentioned and the files are there and are not corrupted. This error fires on seemingly every file that needs to be copied (>190 'file could not copy errors' in my project). Interestingly enough, this issue is not just with `~/bin` files but also errors out on files in the `~/obj` directory as well and occurs for all projects in my solution.

I have tried downgrading to MSBuild 1.6.2 and the problem does not exist there, unfortunately, I cannot stay at version 1.6.2 due to other constraints. 

Versions Information:
- Service Fabric: 6.0.232.9494
- Service Fabric SDK: 2.8.232.9494
- Visual Studio: 2017 v15.5.4

**Comment by dbreshears** [comment]
@keggster101020 ,  you could be  hitting a regression we discovered in the packaging logic of our Microsoft.VisualStudio.Azure.Fabric.MSBuild.1.6.4 Nuget package.      Can you check what the default platform configuration is in the service projects.     In builds after 1.6.2 we are using the default platform specified in the service project rather than the platform specified in the Solution build configuration.     For instance if the default platform is "AnyCPU" in the service project, then to workaround this, edit the .csproj file manually and change to "x64"

```
    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
to
    <Platform Condition=" '$(Platform)' == '' ">x64</Platform>
```

**Comment by keggster101020** [comment]
@dbreshears this was the case, I didn't notice the change as I had changed it awhile back! I have updated my `.csproj` files so in the future they default to x64.

Thanks for the help!

**Comment by dbreshears** [comment]
Great. thanks for confirming.   We are looking into a fix for this.

**Comment by ravipal** [comment]
This is fixed in 1.6.5 version of Microsoft.VisualStudio.Azure.Fabric.MSBuild nuget package.
